packages/micropip/test_micropip.py

```diff
@@ -61,24 +59,34 @@ def test_parse_wheel_url():
 
 def test_install_custom_url(selenium_standalone, web_server_tst_data):
     server_hostname, server_port, server_log = web_server_tst_data
-    selenium_standalone.load_package("micropip")
-    selenium_standalone.run("import micropip")
     base_url = f"http://{server_hostname}:{server_port}/"
     url = base_url + "snowballstemmer-2.0.0-py2.py3-none-any.whl"
-    selenium_standalone.run(f"micropip.install('{url}')")
-    # wait untill micropip is loaded
-    time.sleep(1)
```

Comment:
Great! We can get rid of all the `time.sleep` after using micropip.
There is another one in `src/tests/test_python.py`